cmd/thanos/flags.go

```diff
@@ -201,3 +201,27 @@ func regCommonObjStoreFlags(cmd *kingpin.CmdClause, suffix string, required bool
 		content: bucketConf,
 	}
 }
+
+
+func regCommonTracingFlags(cmd *kingpin.Application, suffix string, required bool, extraDesc ...string) *pathOrContent {
```

Comment:
let's remove `suffix` and `require`: YAGNI - we don't need that now (: same with extraDesc